electroncash_gui/qt/request_list.py

```diff
@@ -155,5 +157,5 @@ def create_menu(self, position):
         menu.addAction(_("Copy URI"), lambda: self.parent.view_and_paste('URI', '', self.parent.get_request_URI(addr)))
         menu.addAction(_("Save as BIP70 file"), lambda: self.parent.export_payment_request(addr))
         menu.addAction(_("Delete"), lambda: self.parent.delete_payment_request(addr))
-        run_hook('receive_list_menu', menu, addr)
+        run_hook('receive_send_via_email_list_menu', menu, addr, self.parent.get_request_URI(addr), self.parent.receive_qr, status) 
```

Comment:
Ok, I suggest we keep the old hook in-place just for excessive caution. We then create a second hook tailored to this plugin. We also pass the "Window" (self.parent) in now, because it bothered me that the callee has to derive that.

```suggestion
        # Original old list menu API, here for legacy reasons
        run_hook('receive_list_menu', menu, addr)
        # New menu API specific to the email plugin
        run_hook('receive_list_menu_for_email_plugin', self.parent, menu, addr,
                 self.parent.get_request_URI(addr), self.parent.receive_qr, status)
```